index.html

```diff
@@ -0,0 +1,257 @@
+      <h2 class="main-program-header light-grey heading-medium">
+        Main Program
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <div class="program-items">
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-book-reader light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Lecture</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fab fa-accusoft light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Exhibition</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="far fa-comments light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Forum</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-code-branch light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Workshop</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-chart-line light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Ignite</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+      </div>
+      <button type="button" name="join-us" class="btn main-btn">
+        Join the Program
+      </button>
+      <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+    </section>
+    <section class="featured-speakers flex-column-center" id="speakers">
+      <h2 class="featured-speakers-header light-black heading-medium">
+        Featured Speakers
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <div class="speakers-list"></div>
+      <button
```

Comment:
-[ ] Great work making the changes requested by the previous reviewers. To make your project complete, kindly implement the final change requested by the previous reviewers The featured speakers section was aligned in two columns when I first reviewed it, now it looks like the changes you made affected this section and it became a single column. I strongly recommend to bring back the previous implementation 👍🏻